app/src/main/java/vn/sunasterisk/music_70/ui/discover/DiscoverFragment.kt

```diff
@@ -49,10 +50,11 @@ class DiscoverFragment : BaseFragment(), GenreContract.View {
         genrePresent = GenrePresenter(tracksRepository, this)
         genrePresent.getTrack(apiTop, false)
         sliderTrackAdapter = SliderTrackAdapter({ tracks, index ->
```

Comment:
```suggestion
        sliderTrackAdapter = SliderTrackAdapter{ tracks, index ->
```

Dùng trailing higher-order function